src/server/scripts/room.js

```diff
@@ -1,5 +1,8 @@
 "use strict";
 
+//TODO shouldnt this be Cards global not Mongo.Collections...
+// Also what does this do? we are on the server do we need to
+// specify what we can and cant do to the cards collection?
```

Comment:
This is allowing the client to directly update the database as opposed to calling a meteor method to do it, this was introduced in https://github.com/Zywave/Retrospectre/pull/23/files
see room.js comment on line 80

So yes, this is some leftover tech debt that needs to be cleaned up. One thing on the the list of things I kept trying to remind myself to fix over break lol